skfda/exploratory/stats/_stats.py

```diff
@@ -91,3 +91,38 @@ def depth_based_median(fdatagrid, depth_method=modified_band_depth):
 
     # The median is the deepest curve
     return fdatagrid[indices_descending_depth[0]]
+
+
+def trim_mean(fdatagrid,
+              proportiontocut,
+              depth_method=modified_band_depth):
+    """Compute the trimmed means based on a depth measure.
+
+    The trimmed means consists in computing the mean function without a
+    percentage of least deep curves. That is, we first remove the least deep
+    curves and then we compute the mean as usual.
+
+    Args:
+        fdatagrid (FDataGrid): Object containing different samples of a
+            functional variable.
+        proportiontocut (float): indicates the percentage of functions to
```

Comment:
There is a problem I have just noticed. In numpy and scipy, `proportiontocut` refers to both sides, while here we only have one side.

Reply:
The thing is that with functional data it is not possible to cut on the other direction right? we are ranking the data according to their depth measure.

Reply:
Right. The problem is the possible confusion for a person used to the scipy `trim_mean`. But the only thing that I can think of is to make a note in the documentation explaining it.

Reply:
Added the note to the documentation. I think that after all we are treating different types of data and the people using this library should be aware. But more documentation never hurts

Reply:
No, I think the note misses the point. The important thing to note here is that the scipy function removes twice the `proportiontocut` while we remove it once.

Reply:
That is true, it does not make sense to remove twice the data in our case as it is unidirectional.